soco/core.py

```diff
@@ -1122,15 +1122,19 @@ def set_relative_group_volume(self, relative_group_volume):
         volume to undershoot the minimum value of 0, the volume will be set
         to 0.
 
+        This method is an alternative to using increment and decrement
+        assignment operators (+=, -=) on the `group_volume` property of a
```

Comment:
+= and -= are in place add and subtract, not increment and decrement

Reply:
True!